app/assets/javascripts/supplemental_files.js

```diff
@@ -119,21 +119,21 @@ $('button[id^=edit_section').on('click', (e) => {
 
   var currentSection = e.target.dataset['sectionId'];
   var isCollapsed = e.target.getAttribute('aria-expanded') == 'true';
-  if (isCollapsed) {
+  if (!isCollapsed) {
     for (var i = 0; i < activeSections.length; i++) {
       if (activeSections[i] == currentSection) {
         activeSections.splice(i, 1);
       }
     }
-  } else {
+  } else if (!activeSections.includes(currentSection)) {
     activeSections.push(currentSection);
   }
   localStorage.setItem('activeSections', JSON.stringify(activeSections));
 });
 
 /* On page reload; collapse sections which were collapsed previously */
-$(document).ready(function () {
-  if (!Modernizr.localStorage) {
+window.addEventListener("load", function () {
```

Comment:
On mco-staging this was running before the `collapse` method had been loaded so was throwing errors.  I tried changing that method to `new bootstrap.Collapse(sectionDiv)` but that also had an issue so I switched it back to `collapse` with the event changed to `load` instead of `ready`.